drake/CMakeLists.txt

```diff
@@ -326,6 +322,12 @@ if (MATLAB_FOUND)
     PROPERTIES TIMEOUT 300)
 endif()
 
+option(BUILD_DOCUMENTATION "Build Doxygen and Sphinx documentation." ON)
```

Comment:
running dot (for doxygen) takes a surprising amount of time on my laptop.  as horrible as it sounds, should this be off by default?